.github/workflows/tox.yml

```diff
@@ -32,7 +32,12 @@ jobs:
     - name: Test with tox
       run: |
         pyenv="py$(echo "${{ matrix.python-version }}" | tr -d '.')"
-        tox -e ${pyenv},${{ matrix.tox-testenv }},flake8,lint
+        tox -e ${pyenv}-${{ matrix.tox-testenv }}
+    - name: Check style
+      if: >-
+        matrix.python-version == 3.9 &&
+        matrix.tox-testenv == 'test'
+      run: tox -e flake8,lint
```

Comment:
IMHO style checks should come before actual tests. Many worst case scenarios can be prevented with that ;-).

Reply:
Ping?

Reply:
It's following what was done before (test run before lint/flake8)